### PR TITLE
added a download app dropdown option

### PR DESCRIPTION
**Pull request by samyadel**
I added an option to download the doghouse application(s) to the dropdown menu you get when you click on your profile

The download icon is an SVG which I downloaded from https://www.fontawesome.com/

### CODE REVIEW

**Comment by overlisted** [comment]
who made the download icon

**Comment by samyadel** [comment]
> who made the download icon

it's from fontawesome: https://fontawesome.com/icons/download?style=solid

**Comment by overlisted** [comment]
@ajmnz what do you think?
image

File: kibbeh/src/ui/SettingsDropdown.tsx [hunk 1]
```diff
@@ -81,6 +82,20 @@ export const SettingsDropdown: React.FC<{
...
             onClick={() => setDebugAudio(!debugAudio)}
           />
+
+            <a
+            href="/download"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <SettingsIcon
+              onClick={onCloseDropdown}
```
Comment (amitojsingh366) [inline kibbeh/src/ui/SettingsDropdown.tsx:92]: why does this not actually take you to the download page?

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/SettingsDropdown.tsx (hunk 1 above)
Comment (samyadel) [reply kibbeh/src/ui/SettingsDropdown.tsx:92]: I ran the web app locally, and everything works fine

Comment (amitojsingh366) [reply kibbeh/src/ui/SettingsDropdown.tsx:92]: does the download button take you to the download page? *(i.e https://dogehouse.tv/download)*

Comment (samyadel) [reply kibbeh/src/ui/SettingsDropdown.tsx:92]: yup

**Comment by amitojsingh366** [comment]
also please use proper commit messages please *(`yarn commit` or `yarn commit:signed`)*

**Comment by ajmnz** [comment]
> @ajmnz what do you think?

@samyadel Copy and paste this one

```jsx
import * as React from "react";

function SvgSolidDownload(props: React.SVGProps<SVGSVGElement>) {
  return (
    <svg
      width={16}
      height={16}
      viewBox="0 0 16 16"
      fill="currentColor"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
    >
      <path d="M12.9001 6.69999C12.4333 4.40003 10.4333 2.66663 8 2.66663C6.06673 2.66663 4.40007 3.76656 3.56673 5.36666C1.56673 5.56669 0 7.26656 0 9.33329C0 11.5332 1.79997 13.3333 4 13.3333H12.6667C14.5 13.3333 16 11.8333 16 9.99996C16 8.23336 14.6335 6.79993 12.9001 6.69999ZM6.93333 8.39996V5.86663H9.06667V8.39996H11.3333L8 11.7333L4.66667 8.39996H6.93333Z" />
    </svg>
  );
}

export default SvgSolidDownload;
```

**Comment by ajmnz** [comment]
And add it to `icons/index.tsx`, don't import directly from the icon file

**Comment by samyadel** [comment]
@ajmnz yup, just did that, and used `yarn commit`

**Comment by amitojsingh366** [comment]
> @ajmnz yup, just did that, and used `yarn commit`

all of your commits should be with `yarn commit`

**Comment by BronzW** [comment]
Could you add translation keys?

**Comment by samyadel** [comment]
> Could you add translation keys?

what?

**Comment by BronzW** [comment]
> > Could you add translation keys?
> 
> what?

read CONTRIBUTING.md. I added a guide for adding translation support for your components. Hope it will be helpful.

**Comment by amitojsingh366** [comment]
trying to rename the commits in here is a big mess, ill create a new pr and add translations to it aswell

**Pull request closed without merging**